src/Rules.Framework/Builder/IRuleBuilder.cs

```diff
@@ -1,6 +1,7 @@
-namespace Rules.Framework.Builder
+namespace Rules.Framework
```

Comment:
Since you have moved the extension methods to this interface and to `RuleBuilder<TContentType, TConditionType>`, changing the namespace was no longer necessary. Or do you have another idea of still changing this namespace? (I just want to understand, that's all 😄)

Reply:
I thought I wouldn't be able to see the extension methods without the direct reference to the Rules.Framework.Builder.

I guess because the `NewRule` returns the `IRuleBuilder<TContentType, TConditionType>` (which is already in the namespace Rules.Framework.Builder) I can see all the methods in that class.

![image](https://github.com/Farfetch/rules-framework/assets/23389613/c43b62a0-8a33-44c6-bc89-e4ad52aa8165)

I've changed the namespaces of the three interfaces (`IRulesBuilder`, `IRootConditionNodeBuilder`, `IFluentComposedConditionNodeBuilder`) back to Rules.Framework.Builder. 👍🏼

Reply:
Yup, that's exactly how you described, all it takes is the initial `RulesBuilder` to be in the `Rules.Framework` namespace 👌